DSharpPlus/Clients/DiscordClient.cs

```diff
@@ -25,19 +31,36 @@ namespace DSharpPlus;
 /// </summary>
 public sealed partial class DiscordClient : BaseDiscordClient
 {
-    #region Internal Fields/Properties
+    internal static readonly DateTimeOffset discordEpoch = new(2015, 1, 1, 0, 0, 0, TimeSpan.Zero);
+    private static readonly ConcurrentDictionary<ulong, SocketLock> socketLocks = [];
+
+    private readonly ConcurrentDictionary<Type, AsyncEvent> events = [];
 
     internal bool isShard = false;
     internal IMessageCacheProvider? MessageCache { get; }
+    private readonly IClientErrorHandler errorHandler;
 
     private List<BaseExtension> extensions = [];
-    private StatusUpdate status = null;
+    private StatusUpdate? status = null;
 
-    private ManualResetEventSlim ConnectionLock { get; } = new ManualResetEventSlim(true);
+    private int heartbeatInterval;
+    private DateTimeOffset lastHeartbeat;
+    private int skippedHeartbeats = 0;

```

Comment:
Why is this explicitly initiated to it's default value?

Reply:
ask emzi, i can remove it

Reply:
Something you'll have to ask Emzi...

Reply:
Just remove it, it's fine